TwilioStore.Services/NotificationService.cs

```
@@ -0,0 +1,48 @@
+﻿using Twilio;
+using TwilioStore.Interfaces.Exceptions;
+using TwilioStore.Interfaces.Services;
+
+namespace TwilioStore.Services
+{
+    public class NotificationService : INotificationService
+    {
+        private readonly TwilioRestClient _client;
+        private readonly INotificationConfiguration _config;
+
+        public NotificationService(INotificationConfiguration config = null)
+        {
+            _config = config ?? new NotificationConfiguration();
+            _client = new TwilioRestClient(_config.AccountSid, _config.AuthToken);
+        }
+
+        public void SendText(string to, string message)
+        {
+            var result = _client.SendMessage(_config.DefaultFromPhoneNumber, to, message);
+            if (result.RestException != null)
+            {
+                throw new NotificationException(result.RestException.Message);
```

💯 for let the application to fail loudly in case of an exception.

> Errors should never pass silently
> -- The Zen of Python --